subcommands/devices/list.go

```diff
@@ -105,6 +106,7 @@ func init() {
 	listCmd.Flags().IntVarP(&deviceInactiveHours, "offline-threshold", "", 4, "List the device as 'OFFLINE' if not seen in the last X hours")
 	listCmd.Flags().StringVarP(&deviceUuid, "uuid", "", "", "Find device with the given UUID")
 	listCmd.Flags().StringSliceVarP(&showColumns, "columns", "", defCols, "Specify which columns to display")
+	listCmd.Flags().IntVarP(&showPage, "page", "p", 1, "Page of devices to display when pagination is needed")
```

Comment:
Maybe, let's allow a user to specify a limit too (e.g. with tail I'm rarely interested in more than top 10-20 results, many users would think the same about this command) ?
It's not about performance, it's about convenience.

Reply:
good idea - https://github.com/foundriesio/fioctl/pull/93/commits/41c5dd8f190b21cb60fd2ac7866d96a0c34d1cfa